### PR TITLE
Set up military history page

### PR DESCRIPTION
**Pull request by cehsu**
These changes set up the military history page.

[Prototype link](https://adhoc.invisionapp.com/share/9MFXXQTUGDK#/screens/282992021)

Some additional adjustments to the common array page were made here. Feedback on this approach and any alternative proposals especially welcome.

TODO: 
- [ ] write tests

screen shot 2018-04-13 at 1 34 51 pm

screen shot 2018-04-13 at 1 37 43 pm

### CODE REVIEW

**Review by peggygannon — COMMENTED** [review]
@cehsu Please see below for my edits to this screen: 

please update the first sentence to:  
This is the service history we have on file for you. If you need to update your service history, you can edit or add another service period.

**Comment by cehsu** [comment]
Thanks @peggygannon, updated.
screen shot 2018-04-13 at 4 40 03 pm

**Comment by peggygannon** [comment]
@cehsu OK, looks good.

**Comment by cvalarida** [comment]
Heads up: Looks like unit tests are failing.

File: src/js/common/schemaform/fields/ArrayField.jsx [hunk 1]
```diff
@@ -136,8 +136,9 @@ export default class ArrayField extends React.Component {
           ? false
           : val;
       });
+      const editingState = this.props.uiSchema['ui:options'].reviewMode;
       const newState = _.assign(this.state, {
-        editing: newEditing.concat(false)
+        editing: newEditing.concat(editingState)
```
Comment (cvalarida) [inline src/js/common/schemaform/fields/ArrayField.jsx]: What problem does this solve?

File: src/js/disability-benefits/526EZ/config/form.js [hunk 2]
```diff
@@ -78,593 +87,662 @@ const formConfig = {
...
-          title: (formData, { pagePerItemIndex }) => _.get(`disabilities.${pagePerItemIndex}.diagnosticText`, formData),
-          path: 'supporting-evidence/:index/evidence-type',
-          showPagePerItem: true,
-          arrayPath: 'disabilities',
-          uiSchema: {
-            disabilities: {
+            servicePeriods: {
+              'ui:title': 'Military service history',
+              'ui:description':
+                'This is the service history we have on file for you. If you need to update your service history, you can edit or add another service period.',
```
Comment (cvalarida) [inline src/js/disability-benefits/526EZ/config/form.js]: I suppose the can also remove a service period too, can't they? :man_shrugging:

File: src/js/disability-benefits/526EZ/config/form.js [hunk 3]
```diff
@@ -78,593 +87,662 @@ const formConfig = {
...
-              disabilities: {
-                type: 'array',
-                items: {
-                  type: 'object',
-                  properties: {}
-                }
-              }
+        }
+      },
+      chapterTwo: {
```
Comment (cvalarida) [inline src/js/disability-benefits/526EZ/config/form.js]: I suppose it probably doesn't matter, but this isn't the right level for these chapters; these are still nested in the `reviewVeteransDetails` chapter.

**Review by cvalarida — APPROVED** [review]
I'm not sure why the change to `ArrayField`, but it looks alright.

:shipit:

File: src/js/common/schemaform/fields/ArrayField.jsx (hunk 1 above)
Comment (cehsu) [reply src/js/common/schemaform/fields/ArrayField.jsx]: Editing state is now configurable, as this view needs to display the items as not editing, vs editing, by default.

**Pull request merged**